### PR TITLE
fix for https://github.com/pvlib/pvlib-python/issues/102

### PR DESCRIPTION
**Pull request by dacoex**
inserted link to variables

### CODE REVIEW

**Comment by wholmgren** [comment]
Thanks. Perhaps I wasn't clear enough in #102... Since you seem particularly interested in improving the documentation, I recommend that you set up readthedocs.org to render your fork of pvlib. This will make it easier to review this PR and future PRs. If you're just making small edits to the docs then I don't think it's needed, but here it will be helpful to see the rendered version since you're making new files and tables.

It is pretty simple to make an account on readthedocs, hook it up to your github account, and then tell rtd to render your fork and possibly your ``doc_var`` branch. See here:

http://docs.readthedocs.org/en/latest/getting_started.html#import-your-docs

Then post a comment here with a link to your version of the new docs.

**Comment by dacoex** [comment]
See this:
http://pvlib-python-dacoex.readthedocs.org/en/latest/

**Comment by wholmgren** [comment]
Thanks, that is helpful. A few comments:

1. The page name is "Variables and style rules" but it currently only has a table of variables. We need to either just have the table of variables and change the name or we need to include some text about style rules and keep the name as is. I don't have an opinion on what's best.
2. I'd replace "rules" with "conventions".
2. Do we need to keep the "comments" column? I'd say no.
2. Do we want to keep the wiki page after merging this? I'd say delete it so that we don't have to make these changes in two places.

**Comment by dacoex** [comment]
OK. 1 question to my earlier comment :+1: 

>Additionally, the csv values can be reused, e.g. to assign variables or translate variables into >descriptions.

We could add the table to the folder
https://github.com/pvlib/pvlib-python/tree/master/pvlib/data

Then, we could read the variables in and make descriptive lables for graphs etc from it:

```
long_name = variable_csv_file['dni']
```

**Comment by wholmgren** [comment]
> We could add the table to the folder
https://github.com/pvlib/pvlib-python/tree/master/pvlib/data Then, we could read the variables in and make descriptive lables for graphs etc from it: long_name = variable_csv_file['dni']

Maybe. I doubt that many people would take advantage of something like that, but it probably doesn't hurt anything. We'd probably want to change the multiple-valued cells if we did this.

**Comment by dacoex** [comment]
@wholmgren I will move the table 
https://github.com/pvlib/pvlib-python/tree/master/pvlib/data

this makes renaming to the agreed variables easier.

**Comment by wholmgren** [comment]
@dacoex you should rebase this and rebuild your docs now that #108 has been merged.

File: .gitignore [hunk 1]
```diff
@@ -69,7 +69,7 @@ coverage.xml
 
 
 #Datafiles
-*.csv
+#*.csv
```
Comment (wholmgren) [inline .gitignore]: No need to change this.

File: docs/sphinx/source/index.rst [hunk 2]
```diff
@@ -42,6 +42,8 @@ and
 
 The GitHub wiki also has a page on `Projects and publications that use pvlib python <https://github.com/pvlib/pvlib-python/wiki/Projects-and-publications-that-use-pvlib-python>`_ for inspiration and listing of your application.
 
+There is a :ref:`convention of variable naming <variables_style_rules>` to ensure consistency throughout the library.
```
Comment (wholmgren) [inline docs/sphinx/source/index.rst]: convention of variable naming --> variable naming convention

File: docs/sphinx/source/variables_style_rules.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+   :stub-columns: 1
+   
+For an definition and further help on symbols and units refer to the following sources:
+
+
+* `Reference Variable List by PVPMC <https://pvpmc.sandia.gov/resources/variable-list/>`_
+* with technical focus in `IEC 61724:1998 -- Photovoltaic system performance monitoring - Guidelines for measurement, data exchange and analysis <https://webstore.iec.ch/publication/5733>`_ ; the Indian Standard referencing the global IEC standard is available online: `IS/IEC 61724 (1998) <https://law.resource.org/pub/in/bis/S05/is.iec.61724.1998.pdf>`_
+* with physics focus:
+
+  * Explanation of Solar irradiation and solar geometry by `SoDa Service <http://www.soda-pro.com/home>`_ and partner pages (http://www.soda-is.com/, http://pro.soda-is.com/)
```
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: Why do you include the "partner pages"? They both request that you just use soda-pro.com

File: docs/sphinx/source/variables_style_rules.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+There is a convention on consistent variable names throughout the library:
+
+.. csv-table:: List of used Symbols and Variables
+   :file: ../../../pvlib/data/variables_style-rules.csv
+   :delim: ;
+   :header-rows: 1
+   :widths: 5, 5
+   :stub-columns: 1
+   
+For an definition and further help on symbols and units refer to the following sources:
```
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: Probably need to say something about the fact that these references might not use the same terminology as us.

File: docs/sphinx/source/variables_style_rules.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+
+* `Reference Variable List by PVPMC <https://pvpmc.sandia.gov/resources/variable-list/>`_
+* with technical focus in `IEC 61724:1998 -- Photovoltaic system performance monitoring - Guidelines for measurement, data exchange and analysis <https://webstore.iec.ch/publication/5733>`_ ; the Indian Standard referencing the global IEC standard is available online: `IS/IEC 61724 (1998) <https://law.resource.org/pub/in/bis/S05/is.iec.61724.1998.pdf>`_
```
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: I don't see the difference between the "technical focus" and many of the "physics focus" pages. I recommend making a single list for simplicity.

**Comment by wholmgren** [comment]
@dacoex thanks for the updates. Please see the line notes above.

File: docs/sphinx/source/variables_style_rules.rst (hunk 3 above)
Comment (dacoex) [inline docs/sphinx/source/variables_style_rules.rst]: The partner pages have extend lists.

**Comment by dacoex** [comment]
all proposals where I did not add a counter comment above are noew implemented.

Probably left with the table wrap issue.

File: pvlib/data/variables_style-rules.csv [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+v_mp, i_mp, p_mp;module voltage, current, power at the maximum power point
+v_oc;open circuit module voltage
+i_sc;short circuit module current
+i_x, i_xx;Sandia Array Performance Model IV curve parameters
+effective_irradiance;effective irradiance
+photocurrent;photocurrent
+saturation_current;diode saturation current
+resistance_series;series resistance
+resistance_shunt;shunt resistance
+tf;transposition factor, the gain ration of the radiation in inclinded plane to horizontal plane: :math:`\frac{poa\_global}{ghi}`
```
Comment (wholmgren) [inline pvlib/data/variables_style-rules.csv]: multiple typos in this line

Comment (wholmgren) [inline pvlib/data/variables_style-rules.csv]: I prefer ``transposition_factor`` to ``tf``.

File: docs/sphinx/source/variables_style_rules.rst [hunk 7]
```diff
@@ -0,0 +1,29 @@
+.. _variables_style_rules:
+
+Variables and Symbols
+=====================
+
+There is a convention on consistent variable names throughout the library:
+
+.. csv-table:: List of used Symbols and Variables
```
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: There are no symbols in this table

File: docs/sphinx/source/variables_style_rules.rst [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+   :file: ../../../pvlib/data/variables_style-rules.csv
```
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: The file name should just use underscores.

File: docs/sphinx/source/variables_style_rules.rst [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+* Explanation of Solar irradiation and solar geometry by `SoDa Service <http://www.soda-pro.com/home>`_
+  
+   * `Acronyms, Terminology and Units <http://www.soda-pro.com/help/general/acronyms-terminology-and-units>`_
+   * `Plane orientations and radiation components <http://www.soda-pro.com/help/general/plane-orientations-and-radiation-components>`_
+   * `Time references <http://www.soda-pro.com/help/general/time-references>`_
+   * `Units and conversion tool <http://www.soda-is.com/eng/education/units.html>`_
+   * `Terminology: definitions of the main quantities. <http://www.soda-is.com/eng/education/terminology.html>`_
+   * `Acronyms in solar radiation <http://www.soda-is.com/eng/education/acronymes.html>`_ (more extensive list)
+
+.. note:: These further references might not use the same terminology as :mod:pvlib. But the physical process referred to is the same.
```
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: The :mod: isn't rendering right. I don't think you need it anyways.

File: docs/sphinx/source/variables_style_rules.rst (hunk 8 above)
Comment (dacoex) [inline docs/sphinx/source/variables_style_rules.rst]: Why?

File: docs/sphinx/source/variables_style_rules.rst (hunk 9 above)
Comment (dacoex) [inline docs/sphinx/source/variables_style_rules.rst]: I guess this note here responds to:

>Probably need to say something about the fact that these references might not use the same terminology as us.

File: docs/sphinx/source/variables_style_rules.rst (hunk 8 above)
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: simplicity

File: docs/sphinx/source/variables_style_rules.rst (hunk 9 above)
Comment (wholmgren) [inline docs/sphinx/source/variables_style_rules.rst]: We don't need :mod:. We do need the note.

Comment (dacoex) [inline docs/sphinx/source/variables_style_rules.rst]: just pushed corrections

**Comment by wholmgren** [comment]
Doesn't look like it's rendering right now.

http://pvlib-python-dacoex.readthedocs.org/en/latest/variables_style_rules.html

**Comment by dacoex** [comment]
looks good now, I hope. CU tomorrow. I am in another tz (UTC+1)

**Comment by wholmgren** [comment]
Looks good, thanks.

**Comment by dacoex** [comment]
Thanks for the patience. I can only learn from this peer review. And it increases my confidence in the code.

Next time tell me and I will stash the trial and error together.

**Pull request merged**